### PR TITLE
Audit logs

### PR DESCRIPTION
**Pull request by kaydee0502**
+ Changes:
    - Added `audited` gem.
    - Setup `audited` table.
    - Enabled `audited` for `Groups` and `GroupMembers` table.
    - Codeclimate configure.

[Notion Task Link](https://www.notion.so/Backend-eng-101-1408fab49eb0425293c17cceced495b5)

### CODE REVIEW

File: devsnest/db/schema.rb [hunk 1]
```diff
@@ -43,6 +65,18 @@
     t.index ["unique_id"], name: "index_contents_on_unique_id"
   end
 
+  create_table "frontend_contents", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb3", force: :cascade do |t|
```
Comment (vedansh) [inline devsnest/db/schema.rb:68]: why are we doing this?

Comment (kaydee0502) [reply devsnest/db/schema.rb]: oops, fixed this just now.

**Review by vedansh — APPROVED** [review]

**Comment by vedansh** [comment]
Let's squash these changes

**Comment by kaydee0502** [comment]
> Let's squash these changes

**Pull request merged**